src/main/java/duke/exception/EmptyInputException.java

```diff
@@ -0,0 +1,15 @@
+package duke.exception;
+
+/**
+ * The Empty input exception.
+ */
+public class EmptyInputException extends DukeException {
```

Comment:
It's nice you separated out this exception instead of also classifying empty input as a DukeException